### PR TITLE
templates: Add assigning the GH issue and Jira card to the next person

### PR DESCRIPTION
**Pull request by travier**
Last release issues where not Jira'd thus I forgot about them during planning.

### CODE REVIEW

File: ISSUE_TEMPLATE/next.md [hunk 1]
```diff
@@ -92,6 +92,6 @@ NOTE: In the future, most of these steps will be automated.
 
 ## Housekeeping
 
-- [ ] If one doesn't already exist, [open an issue](https://github.com/coreos/fedora-coreos-streams/issues/new?labels=kind/release,jira&template=next.md) in this repo for the next release in this stream. Use the approximate date of the release in the title.
+- [ ] If one doesn't already exist, [open an issue](https://github.com/coreos/fedora-coreos-streams/issues/new?labels=kind/release,jira&template=next.md) in this repo for the next release in this stream. Use the approximate date of the release in the title. Set the `jira` label to help with tracking and planning.
```
Comment (jlebon) [inline ISSUE_TEMPLATE/next.md:95]: WDYT about also adding:

> Assign the next person in the rotation to the GitHub and Jira cards.

?

**Review by jlebon — APPROVED** [review]
Good idea, thanks!

**Comment by bgilbert** [comment]
The `open an issue` links already include `jira` in the `labels` list.  Is that not working?

**Comment by jlebon** [comment]
> The `open an issue` links already include `jira` in the `labels` list. Is that not working?

Ahh yup, good catch. It does. Maybe the issue is more that those Jira cards aren't getting assigned and moved into a sprint.

File: ISSUE_TEMPLATE/next.md (hunk 1 above)
Comment (travier) [reply ISSUE_TEMPLATE/next.md:95]: Good idea too.

**Comment by travier** [comment]
> The `open an issue` links already include `jira` in the `labels` list. Is that not working?

Indeed. For some reason the issues this time were not synced to Jira nor assigned to me. I don't know what happened.

**Comment by travier** [comment]
Updated to address feedback

**Review by jlebon — APPROVED** [review]

**Review by bgilbert — APPROVED** [review]
To reduce confusion, you could say something like "This will automatically create a Jira card.  Assign...".  This is still an improvement though!

**Comment by travier** [comment]
Updated to make it an independent bullet point and add a link to the release rotation hackmd.

**Review by bgilbert — APPROVED** [review]

**Pull request merged**